app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">Acme Bank</string>
+    <string name="app_name">Acme Bank - Kotlin</string>
```

Comment:
This is just to not get confused in the Android device when testing many different apps